cpp/ql/src/change-notes/2022-09-05-cleartext-queries.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Modernizations from "Cleartext storage of sensitive information in buffer" (`cpp/cleartext-storage-buffer`) have been ported to the "Cleartext storage of sensitive information in file" (`cpp/cleartext-storage-file`), "Cleartext transmission of sensitive information" (`cpp/cleartext-transmission`) and "Cleartext storage of sensitive information in an SQLite database" (`cpp/cleartext-storage-database`) queries. These changes may reduce false positive results.
```

Comment:
This PR also overrides `allowImplicitRead`, which might give _more_ results. Should the possibility of "more results" also be mentioned in the change-note?

Reply:
Done.